python_on_whales/components/buildx/cli_wrapper.py

```diff
@@ -501,16 +504,27 @@ def disk_usage(self):
         """Not yet implemented"""
         raise NotImplementedError
 
-    def inspect(self, x: Optional[str] = None) -> Builder:
+    def inspect(
+        self,
+        x: Optional[str] = None,
+        bootstrap: Optional[bool] = False,
```

Comment:
```suggestion
        bootstrap: bool = False,
```
`None` is not an acceptable value, as such, it's not an optional.

Reply:
Fixed !